components/mpas-ocean/bld/build-namelist

```diff
@@ -72,6 +72,7 @@ OPTIONS
      -ninst_ocn            NINST_OCN for this case
      -ocn_tidal_mixing     variable for defining if to run with parameterized tidal mixing
                            Options are: false, true. Default is false
+     -glc_nzoc             number of z-ocean classes for indirect glc-ocn coupling [0 | 4 | 30]
```

Comment:
Same question as above: is this restricted to only these 3 values?

Reply:
See my longer response to a related question below.  I followed the implementation of the land multiple elevation classes, which hardcodes a small number of options for the number of levels, and each one corresponds to a preset array of levels.  There is functionality (in land MECs and also my new ocean z-levels) to initialize to a custom array of levels, but there is not at present a way to read them in from file.  That would need to be added in the future if we wanted to support that level of generality.